### DISCUSSION TITLE
2.17 Tech Preview Update?

### DISCUSSION THREAD

**Discussion by johnstonian** [adobe/aem-core-wcm-components#1753] [Q&A] [answered]
I noticed the latest release is 2.17.8, but it appears 2.17 is a "tech preview" and should not be used in production environments. How will the community know when the latest components are ready for production environments?

Thank you!

**Comment by gabrielwalt** [comment, marked as answer]
Hi!
When then new components will be ready to use, we'll release a 3.0 version to communicate clearly that the new components are ready to use.
Best

**Comment by msagolj** [comment]
Maybe also to clarify, 2.17 itself is not a tech preview , 2.17.8 is a normal release that is used in production, all the component versions not marked as WIP in their README ( e.g. [Image V3](https://github.com/adobe/aem-core-wcm-components/tree/master/content/src/content/jcr_root/apps/core/wcm/components/image/v3/image#image-v3-work-in-progress-do-not-use-in-production)) can be used as before.
